Kfilter/src/main/java/com/isaacudy/kfilter/KfilterView.kt

```diff
@@ -474,8 +516,16 @@ class KfilterView @JvmOverloads constructor(context: Context,
 
         override fun onScroll(e1: MotionEvent, e2: MotionEvent, distanceX: Float, distanceY: Float): Boolean {
             val distance = (e1.x - e2.x) / surfaceWidth
+
+            if (e1.x > e2.x) {
+                test(true, e1.x, e2.x)
+            } else {
+                test(false, e1.x, e2.x)
+            }
```

Comment:
I would prefer this if statement to be simplified as follows: 

```kotlin
val isLeft = e1.x > e2.x
test(isLeft, e1.x, e2.x)
```

Reply:
done